app/upgrades.go

```diff
@@ -181,73 +95,10 @@ func GetUpgradeStoreLoader(app *App, info upgradetypes.Plan) baseapp.StoreLoader
 	return upgradetypes.UpgradeStoreLoader(info.Height, &storeUpgrades)
 }
 
-func UpgradeICA(ctx sdk.Context, app *App, versionMap *module.VersionMap) {
-	app.Logger().Info("Initializing ICA")
-
-	// Set the consensus version so InitGenesis is not ran
-	// We are configuring the module here
-	(*versionMap)[icatypes.ModuleName] = app.mm.Modules[icatypes.ModuleName].ConsensusVersion()
-
-	// create ICS27 Controller submodule params
-	controllerParams := icacontrollertypes.Params{
-		ControllerEnabled: false,
-	}
-
-	// create ICS27 Host submodule params
-	hostParams := icahosttypes.Params{
-		HostEnabled:   true,
-		AllowMessages: []string{},
-	}
-
-	// initialize ICS27 module
-	icamodule, correctTypecast := app.mm.Modules[icatypes.ModuleName].(ica.AppModule)
-	if !correctTypecast {
-		panic("mm.Modules[icatypes.ModuleName] is not of type ica.AppModule")
-	}
-	icamodule.InitModule(ctx, controllerParams, hostParams)
-	app.Logger().Info("Finished initializing ICA")
-}
-
-func IncreaseMaxCommissions(ctx sdk.Context, app *App) {
-	oneHundredPct := sdk.OneDec()
-	fivePct := sdk.MustNewDecFromStr("0.05")
-	validators := app.StakingKeeper.GetAllValidators(ctx)
-	ctx.Logger().Info("Increasing all validator's max commission to 100% and max change rate to 5%", "count", len(validators))
-	for _, validator := range validators {
-		validator.Commission.MaxRate = oneHundredPct
-		// Note: This MaxChangeRate bump was added after paua-rc1 was run on testnet.
-		// So, even though it's called by the paua-rc1 upgrade handler now,
-		// it wasn't part of the actual paua-rc1 upgrade that was performed on testnet.
-		if validator.Commission.MaxChangeRate.LT(fivePct) {
-			validator.Commission.MaxChangeRate = fivePct
-		}
-		app.StakingKeeper.SetValidator(ctx, validator)
-	}
-}
-
-func IncreaseMaxGas(ctx sdk.Context, app *App) {
-	ctx.Logger().Info("Increasing max gas per block to 120,000,000")
-	params := app.GetConsensusParams(ctx)
-	params.Block.MaxGas = 120_000_000
-	app.StoreConsensusParams(ctx, params)
-}
-
-func UndoMaxGasIncrease(ctx sdk.Context, app *App) {
-	ctx.Logger().Info("Setting max gas per block back to 60,000,000")
-	params := app.GetConsensusParams(ctx)
-	params.Block.MaxGas = 60_000_000
-	app.StoreConsensusParams(ctx, params)
-}
-
-func SetSanctionParams(ctx sdk.Context, app *App) error {
-	ctx.Logger().Info("Setting sanction params")
-	params := &sanction.Params{
-		ImmediateSanctionMinDeposit:   sdk.NewCoins(sdk.NewInt64Coin("nhash", 1_000_000_000_000_000)),
-		ImmediateUnsanctionMinDeposit: sdk.NewCoins(sdk.NewInt64Coin("nhash", 1_000_000_000_000_000)),
-	}
-	err := app.SanctionKeeper.SetParams(ctx, params)
-	if err != nil {
-		return fmt.Errorf("could not set sanction params: %w", err)
+// RemoveIsSendEnabledEntries removes all entries in the bankkeepers send enabled table
+func RemoveIsSendEnabledEntries(ctx sdk.Context, app *App) {
+	sendEnabledItems := app.BankKeeper.GetAllSendEnabledEntries(ctx)
```

Comment:
This is probably fine ... but should we verify that there is a marker associated with this denom before removing the entry just to be extra careful?